.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: c
 script: asdf plugin-test golang .
 before_script:
+  - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install -y bsdmainutils; fi
```

Comment:
`column` is being used in the `asdf plugin-test` script. Unfortunately the `ppc64le` and `arm64` variants of Ubuntu Xenial do not install `bsdmainutils` package by default.

I have lodged a [question](https://travis-ci.community/t/missing-bsdmainutils-in-ubuntu-ibm-5-4-0-6ubuntu1-16-04-12-build/7013) on Travis CI forum, if they are to agree on bundling `bsdmainutils` on these architectures, we could remove this line